src/dependency-container.ts

```diff
@@ -36,7 +37,9 @@ export const typeInfo = new Map<constructor<any>, ParamInfo[]>();
 
 /** Dependency Container */
 class InternalDependencyContainer implements DependencyContainer {
-  private _registry = new Registry();
```

Comment:
Should have probably done this in a separate PR. But the leading `_` for private fields isn't really recommended by anyone these days. removing